services/submission.js

```diff
@@ -306,7 +306,37 @@ class Submission {
         }
         return result;
     }
-
+    /**
+     * API to export dataRecords of the submission to tsv file by async process
+     * @param {*} params 
+     * @param {*} context 
+     * @returns AsyncProcessResult
+     */
+    async exportSubmission(params, context) {
+        verifySession(context)
+            .verifyInitialized()
+            .verifyRole([ROLES.ADMIN, ROLES.CURATOR]);
+        const aSubmission = await findByID(this.submissionCollection, params._id);
+        if(!aSubmission){
+            throw new Error(ERROR.INVALID_SUBMISSION_NOT_FOUND)
+        }
+        const userInfo = context.userInfo;
+        let isPermitted = this.userService.isAdmin(userInfo.role)
+        //if not an admin, check if the user is a curator
+        if(!isPermitted){
+            const orgId = aSubmission.organization?._id || userInfo.organization?.orgID;
+            if(!orgId) {
+                throw new Error(ERROR.INVALID_EXPORT_METADATA)
+            }
+            const aOrganization = await this.organizationService.getOrganizationByID(orgId)
+            isPermitted = aOrganization?.conciergeID === userInfo._id;
+            if (!isPermitted) {
```

Comment:
This might be more intuitive.
const permitted = XXX;
if curator role {
  if (![userInfo._id].includes(aOrganization?.conciergeID) or userInfo._id !== aOrganization?.conciergeID) {
  }
}